roles/aap_ocp_install/templates/hub/instance.yaml.j2

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: automationhub.ansible.com/v1beta1
+kind: AutomationHub
+metadata:
+  name: {{ aap_ocp_install_hub['instance_name'] }}
+  namespace: {{ aap_ocp_install_namespace }}
+spec:
```

Comment:
What do you think of templating out the spec, or allowing a user to provide their own for both installs.

Reply:
That is one of the additional PRs I'm planning on doing. I didn't want to make this PR more complicated. I would like to introduce the ability to have many (if not all) parameters for the CR's templated and able to be set by the user through variables.